packages/dev/serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -2216,7 +2218,7 @@ export class _Exporter {
                     }
                     return node;
                 });
-            } else if (babylonNode instanceof Camera) {
+            } else if (babylonNode instanceof TargetCamera) {

```

Comment:
Hmm, we don't have any built-in cameras that are not derived from `TargetCamera`, but someone could create a camera that is directly derived from `Camera`. This will no longer work after this change. Are we okay with that?

Reply:
I'm ok with that, this was indeed a choice i made to keep strong typing. My first version was with a cast in `any `to test the existence of the `rotationQuaternion `property. 
Another choice was to put the rotation property at `Camera `level (as beeing done with `absoluteRotation `which is only used into XR) but i want to avoid changes at core level.
I can get back to this.
`const rotationQuaternion = (<any>babylonCamera).rotationQuaternion;`
and then rollback the test to 
`if (babylonNode instanceof Camera)`
Let me know.